### PR TITLE
Fixed SAIC-291: renamed database config options

### PR DESCRIPTION
**Pull request by realsystem**
Now it is used with prefix db_.

### CODE REVIEW

**Comment by roman-verchikov** [comment]
+1, but we also need to update wiki and other docs with these changes

**Comment by realsystem** [comment]
Sure, I'll do it after merge.

**Comment by roman-verchikov** [comment]
Please rebase

**Comment by realsystem** [comment]
rebase done

File: cfglib.py [hunk 1]
```diff
@@ -200,17 +200,17 @@
...
-    cfg.StrOpt('connection', default='mysql+mysqlconnector',
+    cfg.StrOpt('db_connection', default='mysql+mysqlconnector',
                help='driver for db connection'),
-    cfg.StrOpt('host', default='',
+    cfg.StrOpt('db_host', default='',
                help='compute mysql node ip address'),
-    cfg.IntOpt('port', default='3306',
+    cfg.IntOpt('db_port', default='3306',
                help='port for mysql connection'),
     cfg.StrOpt('database_name', default='',
```
Comment (miarmak) [inline cfglib.py]: Maybe It is better to change this option name too for uniformity? (to `db_name`). 
The same for all option groups.

File: devlab/config.template [hunk 2]
```diff
@@ -42,11 +42,11 @@ password = <src_password>
 tenant = <src_tenant>
```
Comment (miarmak) [inline devlab/config.template]: What about `configs/config.ini` file?

**Comment by roman-verchikov** [comment]
please rebase

**Comment by Settis** [comment]
+1

**Comment by miarmak** [comment]
Looks good. +1

**Comment by mirrorcoder** [comment]
+1

**Pull request merged**